account_parent/models/account.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Steigend IT Solutions
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.addons.decimal_precision as dp
+from odoo import api, fields, models
+
+
+class AccountAccountType(models.Model):
+    _inherit = "account.account.type"
+
+    type = fields.Selection(selection_add=[('view', 'View')])
+
+
+class AccountAccount(models.Model):
+    _inherit = "account.account"
+
+    @api.model
+    def _move_domain_get(self):
+        context = dict(self._context or {})
```

Comment:
You should separate code and parameters.

Resources:

 - Rule #3 here: https://www.odoo.com/fr_FR/slides/slide/10-rules-for-safer-code-398?&
 - https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#no-sql-injection